examples/nextjs/pages/index.tsx

```diff
@@ -25,9 +25,9 @@ const Home: NextPage = () => {
   const [newPassword, setNewPassword] = useState('')
   const accessToken = useAccessToken()
   const { signOut } = useSignOut()
-  const { signUp, ...signUpResult } = useEmailPasswordSignUp(email, password)
-  const { signIn } = useEmailPasswordSignIn(email, password)
-  const { signIn: passwordlessSignIn } = useEmailPasswordlessSignIn(email)
+  const { signUpEmailPassword, ...signUpResult } = useSignUpEmailPassword(email, password)
+  const { emailPasswordSignIn } = useEmailPasswordSignIn(email, password)
+  const { emailPasswordlessSignIn } = useEmailPasswordlessSignIn(email)
```

Comment:
Either we should name the variable name and hooks `sign[Up|In]...` or `...Sign[Up|In]` and I think I would prefer to start with `sign[Up|In]...` just like the current `useSignUpEmailPassword` already uses.

Reply:
Corrected in https://github.com/nhost/nhost/pull/273/commits/22424a5f35246c78d88fa636b560647997216ca7